diff --git a/pebble_tool/sdk/manager.py b/pebble_tool/sdk/manager.py
index d9ebf2a..9b1146b 100644
--- a/pebble_tool/sdk/manager.py
+++ b/pebble_tool/sdk/manager.py
@@ -105,6 +105,7 @@ class SDKManager(object):
             self._install_from_handle(f)
 
     def install_toolchain_from_url(self, url, sdk_version, platform_name):
+        return
         print("Downloading toolchain...")
 
         response = requests.head(url)
@@ -160,6 +161,11 @@ class SDKManager(object):
                 Requirements(sdk_info['requirements']).ensure_satisfied()
                 os.mkdir(os.path.join(self.sdk_dir, sdk_info['version']))
                 t.extractall(path)
+
+            print("Patching waf...")
+            waf_path = os.path.join(path, "sdk-core", "pebble", "waf")
+            subprocess.check_call(["patch", waf_path, "/root/waf.patch"])
+
             venv_path = os.path.join(path, ".venv")
             print("Preparing venv... (this may take a while)")
             subprocess.check_call([sys.executable, "-m", "venv", venv_path])
