diff --git a/pebble_tool/sdk/manager.py b/pebble_tool/sdk/manager.py
index 17b8cb1..a996f4b 100644
--- a/pebble_tool/sdk/manager.py
+++ b/pebble_tool/sdk/manager.py
@@ -250,6 +250,12 @@ class SDKManager(object):
             shutil.move(os.path.join(toolchain_path, 'toolchain-' + platform_name, folder), toolchain_path)
         os.rmdir(os.path.join(toolchain_path, 'toolchain-' + platform_name))
 
+        print(f"Removing x64 build of qemu-pebble...")
+        try:
+            subprocess.run(["find", toolchain_path, "-name", "qemu-pebble", "-type", "f", "-delete"], check=True)
+        except subprocess.CalledProcessError:
+            print("Failed to execute. Continuing...")
+
     def install_remote_sdk(self, version):
         sdk_info = self.request("/v1/files/sdk-core/{}?channel={}".format(version, self.get_channel())).json()
         if 'version' not in sdk_info:
